docs/extending-documentation.md

```diff
@@ -214,23 +214,34 @@ In the code above we have 2 places in where you can do overrides: `./.storybook/
 
 We insert a remarkExtend plugin instance for both the local overrides and the global overrides, and pass the necessary arguments. Then we return the new array of plugins for Mdjs.
 
-In each file you need to add a specifier which section you want to override, as a first step we want to load `lea-tab` and `lea-tab-panel` also in the project:
+In each file you need to add a specifier which section you want to override. As a first step we want to load `lea-tab` and `lea-tab-panel` also in the project. We need to add some imports to the top of the file.
 
-````md
-```
+Use triple backtick \` to create a fenced codeblock ` ``` `, and put inside it:
+
+```md
 ::addMdAfter(':root')
 ```
 
-```js script
```

Comment:
did you try using 4 back ticks? or 3 `~` tildes as a guard around the guard?

Reply:
I tried 4 back ticks, looks very ugly because the storybook md parser doesn't handle it well

Reply:
I will try tildes now

Reply:
Storybook's md parser will take ~~~ as opening and then the ``` as closing :( so also not possible